packages/js/components/package.json

```diff
@@ -77,9 +77,9 @@
 		"@wordpress/a11y": "wp-6.0",
 		"@wordpress/api-fetch": "wp-6.0",
 		"@wordpress/base-styles": "wp-6.0",
-		"@wordpress/block-editor": "^9.8.0",
-		"@wordpress/block-library": "^7.19.0",
-		"@wordpress/blocks": "^11.21.0",
+		"@wordpress/block-editor": "wp-6.0",
+		"@wordpress/block-library": "wp-6.0",
+		"@wordpress/blocks": "wp-6.0",
```

Comment:
It looks like this will downgrade these packages slightly. Are we safe to do this?